src/locales/en/translation.json

```diff
@@ -1164,6 +1164,47 @@
           "text": "<p class=\"govuk-body govuk-!-margin-bottom-2\">Contact the GOV.UK One Login team if you cannot use the app.</p>We'll need to delete your details so you can prove your identity again with your new details.<p class=\"govuk-body govuk-!-margin-bottom-2\"></p><p class=\"govuk-body govuk-!-margin-bottom-2\"><a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"{{url}}\">Contact the GOV.UK One Login team (opens in new tab)</a></p>"
         }
       }
+    },
+    "proveIdentityAnotherTypePhotoId": {
+      "title": "Prove your identity another way",
+      "header": "Prove your identity another way",
+      "content": {
+        "paragraph": "You chose to prove your identity another way.",
+        "subHeading": "What would you like to do?",
+        "formRadioButtons": {
+          "backButtonTextDrivingLicence": "Use your UK passport",
```

Comment:
The context might not seem to match the text, but that's because this text is the option for the other photo id.